README.md

```diff
@@ -241,7 +241,7 @@ FROM foo f0_" .
 WHERE ((f0_.name = "bar" OR f0_.surname = "bar"))
 ```
 
-If you want instead to have more OR conditions separated you can do something like this:
+Instead, if you want instead to have more OR conditions separated you can do something like this:
```

Comment:
You should not repeat the word "instead", so it'd be better to remove the second one